lib/mage/Mage.js

```diff
@@ -706,16 +707,30 @@ Mage.prototype.start = function (cb) {
 	return this;
 };
 
+function showDevelopmentModeWarning(mage) {
+	const log = mage.core.logger.warning;
+
+	if (process.stdout.columns > 45) {
+		log
+			.details(' _____             __  __           _')
+			.details('|  __ \\           |  \\/  |         | |')
+			.details('| |  | | _____   _| \\  / | ___   __| | ___')
+			.details('| |  | |/ _ \\ \\ / | |\\/| |/ _ \\ / _` |/ _ \\')
+			.details('| |__| |  __/\\ \V /| |  | | (_) | (_| |  __/')
+			.details('|_____/ \\___| \\_/ |_|  |_|\\___/ \\__,_|\\___|');
```

Comment:
Have you considered writing straight to stdout instead? Especially since you're checking for terminal width, and the logger could be writing anywhere (syslog, graylog, you name it).

Reply:
Is there a way to target a specific log writer? I would like to avoid having to colorize this myself...

Reply:
Hmm, nevermind...